composer.json

```diff
@@ -43,10 +43,10 @@
         "terminal42/contao-tablelookupwizard": "^3.2 || ^4.0",
         "terminal42/notification_center": "^1.0",
         "terminal42/dc_multilingual": "^4.0",
-        "guzzlehttp/guzzle": "^6.0",
         "tecnickcom/tcpdf": "^6.2.22",
         "patchwork/utf8": "^1.2",
-        "symfony/filesystem": "^3.4 || ^4.0 || ^5.0 || ^6.0"
+        "symfony/filesystem": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/http-client": "^4.4 || ^5.4"
```

Comment:
Can probably also be forward compatible with 6.0?

Reply:
Is it?

Reply:
Looks like: https://github.com/symfony/http-client/blob/6.0/CHANGELOG.md

Reply:
Added in 690e46941.